monasca_agent/collector/checks_d/ceph.py

```diff
@@ -416,8 +428,8 @@ def _get_mon_metrics(self, ceph_status):
         {'monitor1': {metric1': value1, ...}, 'monitor2': {metric1': value1}}
         """
         mon_metrics = {}
-        for health_service in ceph_status['health']['health'][
-                'health_services']:
+        for health_service in ceph_status.get('health').get('health', {}).get(
```

Comment:
Is this information available, but in a different place?

Reply:
Hmm I don't actually know, I was just trying to get it to work at the time without coughing errors iirc.

Reply:
I suppose i could check the monasca agent log on soft iron ceph cluster